src/redux/news/news.sagas.js

```diff
@@ -29,16 +29,63 @@ function* handleNewsLoad() {
                   primary{
                     medium
                   }
+                  additional{
+                    medium
+                  }
                 }
+                video
                }
              }`
     );
     yield put(setNews(news.data.getAllNews));
+    yield put(setLoading(news.loading));
```

Comment:
Where do you take `news.loading`? Does the response contain any loading stuff? I thought loading should be taken from redux state

Reply:
Saga response has data with data and loading indicator that tells us is it true or false